### DISCUSSION TITLE
Access to all session videos

### DISCUSSION THREAD

**Discussion by melatRemarkable** [community/events#24] [Community]
I am so excited to join and be part of this conference. Is it possible to watch all the sessions on my pace since, it is going to be late in my time zone?

**Comment by Youssef1313** [comment]
Probably they will be available at https://watch.githubuniverse.com/on-demand ?

**Comment by queenofcorgis** [comment]
Hey @melatRemarkable - I'm excited you're excited 🎉 All of our core content will be broadcast live and posted on-demand on githubuniverse.com for anyone to watch after the event. Presentations will be available in this repo, post-event.

**Reply by melatRemarkable** [reply]
Thanks, is it like i have to demand all the sessions or will i get a link to watch them whenever i want? how long will it also be available?
